content/realtime/channel-params.textile

```diff
@@ -0,0 +1,91 @@
+
+For example, to specify the parameter @foo@ with value @bar@ on channel @baz@, the qualified channel name would be @[?foo=bar]baz@. If the channel name already has a qualifier, such as @[meta]log@, then the query string follows the existing qualifier, as in @[meta?foo=bar]log@.
+
+In an SSE connection, it is also possible to specify channel parameters as a query string in the connection URL, instead of as a qualifier on an individual channel name. In this case, the given channel parameters apply to all channel attachments associated with that connection.
+
+h2(#rewind). rewind parameter
+
+The @rewind@ channel parameter relates to the initial attachment of a connection to a channel, and expresses the intent to attach to the channel at a position, or a point in time, in the past (that is, effectively "rewinding" the channel for the purposes of the present attachment).
+
+A @rewind@ parameter can express a channel position in terms of a number of messages, or a time interval.
+
+A @rewind@ value that is simply a number @n@ (eg @rewind=1@) is a request to attach to the channel at a position @n@ messages before the present position. If that attachment is successful, and one or more messages exist on the channel prior to the present position, then those messages will be delivered to the subscriber immediately after the attachment has completed, and before any subsequent messages that arise in real time.
+
+If fewer than the requested number of messages exists on the channel (including the case that there are no prior messages), then the available messages are sent; this does not constitute an error.
+
+A @rewind@ value can also be a string that is a time interval specifier. Supported specifier values express an integral number of seconds (eg @15s@), minutes (eg @5m@) or hours (eg @4h@). If that attachment is successful, and one or more messages exist on the channel in the given time interval prior to the present time, then those messages will be delivered to the subscriber immediately after the attachment has completed, and before any subsequent messages that arise in real time.
+
+If fewer than the requested number of messages exists on the channel in that interval (including the case that there are no messages), then the available messages are sent; this does not constitute an error.
+
+If the number of messages within the specified interval is greater than the attachment @limit@ (which defaults to 100), then only the most recent messages up to that limit are sent. The attachment succeeds, but truncation of the message backlog is indicated as a non-fatal error in the attachment response.
+
+It is also possible to specify a different limit via a @limit@ channel parameter. So, for example, to request up to 10 messages, in a window 5m before the present time, specify a channel parameter string of @[?rewind=5m&limit=10]@.
```

Comment:
no it isn't

Reply:
Why not?

Reply:
We've never supported a separate `limit` channel param. I don't recall it ever being mentioned before now. It's certainly not in any of the issues, https://github.com/ably/realtime/issues/2464 or otherwise. I think Paddy may have just gotten the user API confused with the internal ChannelRecoverOptions struct or something.

If we want to support something like that then we'd need to discuss what the api should look like, then implement it. Not entirely obvious, since two different parameters both of which can be a limit might be a bit confusing - eg what would `rewind=10&limit=5` mean?

Reply:
> Not entirely obvious, since two different parameters both of which can be a limit might be a bit confusing 

Sure, but that's clear to document.  Rewind integers are incompatible with limit params, whereas time based ones can have a limit param.  I don't see why this is confusing and I think it makes a lot of sense.